src/command_modules/azure-cli-cognitiveservices/azure/cli/command_modules/cognitiveservices/commands.py

```diff
@@ -26,5 +24,8 @@ def load_command_table(self, _):
         g.command('regenerate', 'regenerate_key')
         g.command('list', 'list_keys')
 
+    with self.command_group('cognitiveservices account usage', mgmt_type) as g:
+        g.command('show', 'get_usages')
```

Comment:
There should not be any command groups with a sole command. The command should just be `show-usage` under the `cognitiveservices account` group. There is much precedence for this in the CLI.

Reply:
@tjprescott I moved it to `cognitiveservices account` group.
I also renamed it from `show-usage` to `list-usages` because the result is a list a usages.